quarkchain/cluster/master.py

```diff
@@ -1804,14 +1804,14 @@ async def get_total_balance(
         branch: Branch,
         block_hash: bytes,
         token_id: int,
-        starter: Optional[bytes],
-        limit: int = 100,
+        starter: bytes,
+        limit: int,
     ) -> Optional[Tuple[int, bytes]]:
         if branch.value not in self.branch_to_slaves:
-            return None
+            raise Exception("Branch not found")
 
         slave = self.branch_to_slaves[branch.value][0]
-        address = Address(starter if starter else bytes(20), branch.value)
+        address = Address(starter, branch.value)
```

Comment:
note `slave.get_total_balance` may also return `None`. in that case it should also be an invalid request failure

Reply:
so we need to add an exception in slave.get_total_balance when the response code is not 0?